code/modules/client/preferences_gear.dm

```diff
@@ -1241,6 +1253,12 @@ var/global/list/gear_datums = list()
 	sort_category = "misc"
 	cost = 1
 
+/datum/gear/cape
```

Comment:
It has a max_storage space of 12, which equates to about three items. Compare this to the backpack's max_storage space of 28.